packages/collector/src/announceCycle/unannounced.js

```diff
@@ -86,19 +92,26 @@ function tryToAnnounce(ctx, retryDelay = initialRetryDelay) {
     try {
       agentResponse = JSON.parse(rawResponse);
     } catch (e) {
-      logger.warn(
-        'Failed to JSON.parse agent response. Response was %s. Will retry in %s ms',
-        rawResponse,
+      logger.error(
```

Comment:
Changed this from warn to error. This should really not happen and it would more often than not be something that will not self-heal when retrying.